documentation/book/assembly-healthchecks.adoc

```diff
@@ -0,0 +1,27 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+// assembly-deployment-configuration-kafka-connect.adoc
+// assembly-deployment-configuration-kafka-connect-s2i.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-healthchecks-{context}']
+
+= Healthchecks
+
+{ProductPlatformName} supports two types of Healthcheck probes:
+
+* Liveness probes
+* Readiness probes
+
+For more details about the probes, see Kubernetes documentation: {K8sLivenessReadinessProbes}.
```

Comment:
Do we want to have the kubernetes conditional around this?

Reply:
I will fix the sentence so that it doesn't refer to Kubernetes. However, we should find the OpenShift alternative for all the `K8s...` links listed in `attributes.adoc`. I can help you with it.